src/python/pants/option/global_options.py

```diff
@@ -65,6 +65,13 @@ def register_bootstrap_options(cls, register):
                       'pants.backend.python',
                       'pants.backend.jvm',
                       'pants.backend.codegen',
```

Comment:
I mean, for code that still does
"from pants.backend.codegen.targets.java_thrift_library import JavaThriftLibrary",
it is an error now?